state-chain/primitives/src/lib.rs

```diff
@@ -322,3 +322,37 @@ fn is_more_recent_semver() {
 	ensure_left_is_more_recent(ver(1, 0, 0), ver(0, 2, 2));
 	ensure_left_is_more_recent(ver(1, 1, 0), ver(1, 0, 2));
 }
+
+pub const MAX_BENEFICIARIES: u32 = 4;
+
+#[derive(Clone, Debug, PartialEq, Eq, Encode, Decode, TypeInfo, Serialize, Deserialize)]
+#[serde(untagged)]
+pub enum BrokerFees<Id> {
+	Single(BasisPoints),
+	Multiple(BoundedVec<Beneficiary<Id>, ConstU32<MAX_BENEFICIARIES>>),
```

Comment:
Does it make sense to allow for the super(?)broker to also specify a commission rate directly without consuming a beneficiary slot?

Reply:
This type exists mainly to allow backwards compatibility in the broker api. It's also more efficient (saves 32 bytes per extrinsic) and therefore cheaper for the user and the network in the base case where you don't want to split fees. This is why we decided to use it in the extrinsic too.

Reply:
I understand, but was wondering if it makes sense to include a separate bps field in the `Multiple` variant that allows the broker opening the channel to specify their own bps without having to consume one of the beneficiary slots